src/main/webapp/resources/js/pages/analysis/components/AnalysisProvenance.jsx

```diff
@@ -170,7 +173,10 @@ export default function AnalysisProvenance() {
   // the user is viewing the provenance for
   function getCreatedByToolPanels() {
     let panels = [];
-    if (analysisOutputsContext.outputs !== null) {
+    if (
+      analysisOutputsContext.outputs !== null &&
```

Comment:
`analysisOutputsContext.outputs !== null` can always be shortened to `!analysisOutputsContext.outputs` bcause `!null === true`

Reply:
Provenance doesn't display if i set it to `!analysisOutputsContext.outputs`